layer/migration_test.go

```diff
@@ -88,12 +93,6 @@ func TestLayerMigration(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	root := filepath.Join(td, "layers")
-	ls, err := newStoreFromGraphDriver(root, graph, runtime.GOOS)
```

Comment:
I moved instantiation of the layer store in this test before any graphdriver operations are performed, so that the cleanup procedure does not interfere with the test.